src/message_interface/dtos.rs

```diff
@@ -113,20 +114,59 @@ impl From<&AddressWithUnspentOutputs> for AddressWithUnspentOutputsDto {
     }
 }
 
-/// Dto for the balance of an account, returned from [`crate::account::handle::AccountHandle::sync()`] and
-/// [`crate::account::handle::AccountHandle::balance()`].
-#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize)]
-pub struct AccountBalanceDto {
+/// Base coin fields for [`AccountBalance`]
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct BaseCoinDto {
```

Comment:
I think I'd call it `BaseCoinBalanceDto` for consistency with `NativeTokensBalanceDto`